database/pgx/pgx.go

```diff
@@ -45,7 +45,8 @@ var (
 type Config struct {
 	MigrationsTable       string
 	DatabaseName          string
-	SchemaName            string
+	SchemaName            string // Private field, computed dynamically
+	TableName             string // Private field, computed dynamically
```

Comment:
Clients shouldn't be able to set this so don't export this. Also, `tableName` isn't as descriptive so let's rename this to: `migrationsTableName`